core/src/main/scala/org/http4s/headers/Referer.scala

```diff
@@ -17,13 +17,13 @@
 package org.http4s
 package headers
 
-import org.http4s.util.Writer
 import java.nio.charset.StandardCharsets
 import org.typelevel.ci.CIString
 
-object Referer extends HeaderKey.Internal[Referer] with HeaderKey.Singleton {
-  override def parse(s: String): ParseResult[Referer] =
-    ParseResult.fromParser(parser, "Invalid Referer")(s)
+object Referer {
+
+  def parse(s: String): ParseResult[Referer] = headerInstance.parse(s)
```

Comment:
Stylistically, I would go the other direction here: make these the `ParseResult.fromParser` calls, and just put `parse` at the third argument in the instance.

Reply:
Ok, I'll change it

Reply:
Resolved